### ISSUE TITLE
[Question] Does this Captcha also works for the initial creation of a new account?

### ISSUE THREAD

**Issue by PeterCodar** [OXIDprojects/captcha-module#9]
In the project description, this 4 forms/cases are listed:

- contact
- invite
- pricealarm
- suggest

Does this module also cover the initial creation of a new account (for new users, especially for spam/fake account users)?

**Comment by gp-itholics** [comment]
No. But you could use this module as base for a custom/own module which could add the feature.
